services/localazy/localazy-langs-simple.service.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+const LocalazyBase = require('./localazy-base.service')
+
+module.exports = class LocalazyLangsSimple extends LocalazyBase {
```

Comment:
I'm personally not a fan of this badge variant. It's almost the same as the `LocalazyLangs` one, but less clear. The word "translated" is not quantifiable when used in isolation, which makes the badge somewhat confusing IMHO.

Maybe merge `LocalazyLangsSimple ` and `LocalazyLangs` and use the noun form instead? 
![](https://img.shields.io/badge/translations-97-blue)

Reply:
Yeah, makes sense. 🙂